tsdfileapi/exc.py

```diff
@@ -199,7 +199,7 @@ def format_component(component: str) -> str:
         code = HTTPStatus.INSUFFICIENT_STORAGE
         status = code.value
         reason = "Project has run out of disk quota"
-        message = ", ".join(code.phrase, reason, details)
+        message = ", ".join([code.phrase, reason, details])
```

Comment:
Eirik pointed out in the issue that this line can actually be removed since the function below does the job right :)

Reply:
Yeah, I have been blind but now I can see. Removed with [d69664b](https://github.com/unioslo/tsd-file-api/pull/276/commits/d69664b0c5023aee10e761d3d82370e7edd62754).